wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -1770,9 +1888,10 @@ type family Update (original :: *) :: * where
   Update WalletAddress = () -- read-only
 
 type family New (original :: *) :: * where
-  New Wallet  = NewWallet
-  New Account = NewAccount
-  New WalletAddress = NewAddress
+  New Wallet         = NewWallet
+  New Account        = NewAccount
+  New WalletAddress  = NewAddress
+  New ExternalWallet = NewExternalWallet
```

Comment:
so this kind of vertical alignment leads to difficult to scan diffs; let's instead go with

```
type family New (original :: *) :: * where
    New Wallet =
        NewWallet
    New Account =
        NewAccount
    -- etc...
```

That way, it is very easy to see what is new at a glance.